Resources/public/Admin.js

```diff
@@ -69,6 +71,44 @@ var Admin = {
             overflow: 'scroll'
         });
     },
+    
+    setup_select: function(subject) {
+       jQuery('select', subject).each(function() {
+            var select = jQuery(this);
+            var fieldContainer = select.closest('div[class="field-container"]');
+            var editBtn = fieldContainer.find('a[class~="sonata_model_edit_btn"]');
+
+            if (editBtn.length) {
+                // save anchor reference
+                editBtn.attr('data-sonata-many-to-one-href', editBtn.attr('href'));
+                editBtn.rewriteBtn = function(id) {
+                    this.removeClass('disabled');
+                    this.attr(
+                        'href',
+                            this.attr('data-sonata-many-to-one-href').replace(
```

Comment:
Hi @rande, what do you think about customizing ``ChoiceView`` and injecting ``RouteGenerator`` in ``ModelType`` ?